src/components/tag/tag.stories.js

```diff
@@ -1,13 +1,53 @@
 import { storiesOf } from '@storybook/vue'
+import Close from 'vue-material-design-icons/Close.vue'
 
 import aposTag from './tag.vue'
+// import aposIcon from '../icon/icon.vue'
 
 storiesOf('Tag', module)
   .add('Standalone', () => ({
     components: { aposTag },
     template: '<aposTag label="Portrait" />'
   }))
   .add('List', () => ({
-    components: { aposTag },
-    template: '<aposTag label="Portrait" />'
+    components: { aposTag, Close },
+    template: `
+      <ul class="apos-tag-list">
+        <li>
+          <aposTag label="Community">
```

Comment:
should we pass some base `modifier`, like "primary" or something so Vue doesn't render an `undefined` fallback?

Reply:
Changed this prop to be optional and for it to be the whole class name. Maybe the prop name should be different? extraClasses?